examples/decomposition/plot_ica_vs_pca.py

```diff
@@ -37,7 +37,7 @@
 
 from sklearn.decomposition import PCA, FastICA
 
-# #############################################################################
+# %%
 # Generate sample data
```

Comment:
Adding an equal length line of hyphens under the text will convert it to a heading in the rendered example. For instance:
```suggestion
# %%
# Generate sample data
# --------------------
```
This may be worth doing for both sections.

Reply:
Thanks @jsilke. This change is consistent with other examples. I have adopted your suggestion.